src/parser.ts

```diff
@@ -28,7 +28,7 @@ export default class MetaEditParser {
         if (!frontmatter) return [];
 
         //@ts-ignore - this is part of the new Obsidian API as of v1.4.1
-        const {start, end} = fileCache?.frontmatterPosition;
+        const {start, end} = fileCache?.frontmatterPosition ?? fileCache?.frontmatter?.position;
```

Comment:
This isn't ideal because if anything's null/undefined then destructuring will throw an exception, but that would happen even before this change.